src/Illuminate/Database/Eloquent/Collection.php

```diff
@@ -351,4 +351,50 @@ public function toBase()
     {
         return new BaseCollection($this->items);
     }
+
+    public function update()
+    {
+        $dirties = [];
+        $dirtyColumns = [];
+        /** @var Model $model */
+        foreach ($this->items as $model) {
+            $dirty = $model->getDirty();
+            if (count($dirty)) {
+                $dirties[$model->getKey()] = $dirty;
+                foreach ($dirty as $column => $value) {
+                    if (! array_key_exists($column, $dirtyColumns)) {
+                        $dirtyColumns[$column] = [];
+                    }
+                    $dirtyColumns[$column][] = [$model->getKey(), $value];
+                }
+
+            }
+        }
+
+        if (! count($dirties)) {
+            return;
+        }
+
+        $keyName = $model->getKeyName();
+        $params = [];
+        $columnUpdates = [];
+        foreach ($dirtyColumns as $column => $rows) {
+            $columnUpdate = "`$column` =  CASE";
+            foreach ($rows as $row) {
+                $columnUpdate .= " WHEN `$keyName` = ? THEN ?";
+                $params[] = $row[0];
+                $params[] = $row[1];
+            }
+            $columnUpdate .= " ELSE `$column` END";
+            $columnUpdates[] = $columnUpdate;
+        }
+
+        $table = $model->newQuery()->getQuery()->getGrammar()->wrapTable($model->getTable());
+        $sql = "UPDATE $table SET ".join(',', $columnUpdates)." WHERE $keyName IN (".join(',', array_fill(0, count($dirties), '?')).')';
```

Comment:
I tested MySQL, and checked the documentations on sqlite. Sounds it is an standard statement. But no I am not sure yet.